search/search-dev-tools/src/components/query/index.js

```diff
@@ -72,9 +72,9 @@ const Query = ( { args, request, url, query_args, backtrace = [] } ) => {
 	}, [ state.query, state.editing ] );
 
 	// Re-highlight the query on result change (after the response is received).
-	useEffect( () => {
-		highlightElement( queryResultRef.current );
-	}, [ queryResultRef, state.result ] );
+	// useEffect( () => {
+	// 	highlightElement( queryResultRef.current );
+	// }, [ queryResultRef, state.result ] );
```

Comment:
I would suggest that we remove the commented-out code - we can always restore it from Git.

Reply:
Also, `highlightElement` on line 5 also needs to be removed.